app/controllers/general.php

```diff
@@ -531,6 +533,40 @@ function router(App $utopia, Database $dbForConsole, SwooleRequest $swooleReques
             }
         }
 
+        // API Token authentication (like API Key but JWT short-term)
```

Comment:
Internally maybe we can call it dynamic key vs the regular key which is persistent.